test_code/AWS_MACHINES/data.tf

```diff
@@ -3,21 +3,23 @@
 #   url = "https://ipv4.icanhazip.com/?4"
 # }
 
-# Use a pre-deployed vpc - this is where ec2 instance will be deployed
-data "aws_vpc" "main" {
-  default = false
-  tags = {
-    Name = "linux-host-test-vpc"
-  }
-}
 
-data "aws_subnet" "main" {
-  vpc_id = data.aws_vpc.main.id
-  filter {
-    name   = "tag:Name"
-    values = ["linux-host-test-subnet"]
-  }
-}
+### As of 08/03, commented out in favor of networking.tf ## 
```

Comment:
nit: If we don't need this, why not just remove it altogether?  If someone needs it in the future, we have it in git to reference.

Reply:
Good point. Let me remove that